src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/leaked-passwords/leakedPasswordsData.tsx

```diff
@@ -0,0 +1,157 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import type { ReactNode } from "react";
+import passwordIllustration from "../images/leaked-passwords.svg";
+import securityQuestionsIllustration from "../images/security-questions.svg";
+import { SubscriberBreach } from "../../../../../../../../utils/subscriberBreaches";
+import { GuidedExperienceBreaches } from "../../../../../../../functions/server/getUserBreaches";
+import { getL10n } from "../../../../../../../functions/server/l10n";
+
+export type LeakedPasswordsContent = {
+  summary: string;
+  description: ReactNode;
+  recommendations?: {
+    title: string;
+    steps: ReactNode;
+    subtitle?: string;
+  };
+};
+
+export type LeakedPasswordsTypes = "password" | "security-questions";
+
+export type LeakedPassword = {
+  type: LeakedPasswordsTypes;
+  title: string;
+  illustration: string;
+  content: LeakedPasswordsContent;
+};
+
+function getLeakedPasswords({
+  dataType,
+  breaches,
+}: {
+  dataType: string;
+  breaches: GuidedExperienceBreaches;
+}) {
+  const l10n = getL10n();
+
+  const findFirstUnresolvedBreach = (
+    breachClassType: "passwords" | "securityQuestions"
```

Comment:
You could just reuse `LeakedPasswordsTypes` here.

```suggestion
    breachClassType: LeakedPasswordsTypes
```